modules/wayprompt/hm.nix

```
@@ -1,35 +1,35 @@
-{ config, lib, ... }:
-let
-  cfg = config.stylix.targets.wayprompt;
+{ mkTarget, lib, ... }:
+mkTarget {
+  name = "wayprompt";
+  humanName = "Wayprompt";
 
-  opacity = lib.toHexString (builtins.ceil (config.stylix.opacity.popups * 255));
-in
-{
-  options.stylix.targets.wayprompt.enable =
-    config.lib.stylix.mkEnableTarget "Wayprompt" true;
+  configElements =
+    { colors, opacity }:
+    let
+      opacity' = lib.toHexString (builtins.ceil (opacity.popups * 255));
+    in
+    {
+      programs.wayprompt.settings.colours = with colors; {
+        background = "${base00-hex}${opacity'}";
```

```suggestion
        background = "${base00}${opacity'}";
```
See https://github.com/nix-community/stylix/pull/1378#discussion_r2106312914